### PR TITLE
feat: add badges to new navigation sidebar (WPB-6651)

### PR DESCRIPTION
**Pull request by thisisamir98**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-6651" title="WPB-6651" target="_blank">TaskWPB-6651</a>  [Web] Add focus and badges to sidebar items
  </td></table>
  <br />
 

image

### CODE REVIEW

File: src/style/common/label.less [hunk 1]
```diff
@@ -22,6 +22,11 @@
   font-weight: @font-weight-semibold;
 }
 
+.label-sm {
+  font-size: @font-size-small;
+  font-weight: @font-weight-semibold;
+}
+
```
Comment (V-Gira) [inline src/style/common/label.less]: I think that's `label-2` in `typing.less` , the new label templates @przemvs  was talking about the other day

Comment (thisisamir98) [reply src/style/common/label.less]: image
gray-70 from .label-2 does not work here

Comment (thisisamir98) [reply src/style/common/label.less]: https://github.com/wireapp/wire-webapp/pull/16849/commits/25cd5b5bce57313f0fdfbbdecea9351b32096ff8

File: src/style/list/conversations.less [hunk 2]
```diff
@@ -89,13 +96,20 @@
...
+    background: var(--red-500);
+    color: var(--button-tertiary-bg);
+    .label-sm;
+  }
+
+  &.active {
+    background-color: var(--gray-95);
+  }
+  .conversations-sidebar-btn--text {
+    color: var(--gray-100);
```
Comment (V-Gira) [inline src/style/list/conversations.less]: That looks, like something from before your last PR, it's duplicated with the correct values below

**Review by V-Gira — APPROVED** [review]

**Review by V-Gira — COMMENTED** [review]
Looking good, just one thing that looks like it should be snipped :scissors:

**Review by V-Gira — APPROVED** [review]

**Pull request merged**